build_tools/cmake/iree_copts.cmake

```diff
@@ -334,15 +334,17 @@ if(CMAKE_CXX_FLAGS AND "${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
   string(REPLACE "/GR" "" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
 endif()
 
+# Find and add threads as dependency.
```

Comment:
I know it's not necessary, but I think it would be nice to explicitly initialize thread deps to empty here, just so that we aren't expanding an undefined variable later. I think modern cmake may actually have a flag to error on undefined variables that we could turn on (IIRC we used to support cmake versions that were too old to enable this)